rosdep/python.yaml

```diff
@@ -1057,6 +1057,11 @@ python-oauth2:
   debian: [python-oauth2]
   fedora: [python-oauth2]
   ubuntu: [python-oauth2]
+python-oauth2client:
+  debian: [python-oauth2client]
+  fedora: [python-oauth2client]
+  ubuntu:
+    trusty: [python-oauth2client]
```

Comment:
Yes, I have read you comment. And as I mentioned e.g. Utopic (14.10) is not listed on the Ubuntu website anymore since it has been EOLed. So looking at the website won't answer the question if it is available. Either look on old-releases.ubuntu.com or use docker to check.